db/migrate/20221218120314_create_feeds.rb

```diff
@@ -0,0 +1,36 @@
+class CreateFeeds < ActiveRecord::Migration[7.0]
+  def change
+    create_table :feeds do |t|
+      t.integer :feed_item_id, null: false
+      t.string :source, limit: 32, null: false
+      t.integer :source_id, null: false
+      t.string :title, null: false
+      t.string :url, null: false
+
+      # In the REAL WORLD, this should be in a
+      # published_feeds join table...
+      #
+      # Whether or not this feed has been published
+      # (emailed) to the other team members; this
+      # will only be true if the feed was recommended
+      # by at least 1 team member.
+      t.boolean :published, default: false, null: false
```

Comment:
Thinking ahead. When we create our "Publish" sidekiq job to send recommended (starred) feeds to our users, this will make things run a little faster.